index.js

```diff
@@ -0,0 +1,35 @@
+import BookStorage from './modules/bookStorage.js';
+import { setupNavigation } from './modules/navigation.js';
+import { DateTime } from './modules/luxon.js';
+
+const form = document.querySelector('.add-form');
+const bookList = document.querySelector('.book-list');
+const date = document.getElementById('date');
+const stringDate = {
+  month: 'long', day: 'numeric', year: 'numeric', hour: 'numeric', minute: 'numeric', hour12: true, second: 'numeric',
+};
+const now = DateTime.now();
+const newDate = now.setLocale('en-US').toLocaleString(stringDate);
+date.innerHTML = newDate;
+
+const storage = new BookStorage();
+
+// Button listeners
+form.addEventListener('submit', (event) => {
+  event.preventDefault();
+  const title = document.querySelector('#title');
+  const author = document.querySelector('#author');
+
+  storage.createBook(title.value, author.value);
+  storage.displayData();
+});
+
```

Comment:
- [ ] Kindly reset the inputs after adding a new book and add a message to notify the user that the book is successfully added ➕ ➕ ➕ It will improve the UX of your web application.